swiftnav_console/main.py

```diff
@@ -478,9 +478,16 @@ def _process_message_buffer(self, buffer):
 
             up = m.statusBarStatus.ntripUpload
             down = m.statusBarStatus.ntripDownload
+            down_units = "B/s"
+
+            if down >= 1000:
+                down /= 1000
+                down = round(down, 1)
+                down_units = "KB/s"
+
```

Comment:
upload bytes is roughly 80bytes, download is the only value that needs the conversion sometimes